docs/docs/components/reanimated-drawer-layout.mdx

```diff
@@ -0,0 +1,193 @@
+---
+id: reanimated-drawer-layout
+title: Reanimated Drawer Layout
+sidebar_label: Reanimated Drawer Layout
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Cross-platform replacement for the React Native's [DrawerLayoutAndroid](http://reactnative.dev/docs/drawerlayoutandroid.html) component.  
+Provides a compatible API while allows for the component to be used on both Android and iOS.  
+Please refer to [React Native docs](http://reactnative.dev/docs/drawerlayoutandroid.html) for the detailed usage for standard parameters.
+
+### Usage:
+
+To use it, import it in the following way:
+
+```js
+import ReanimatedDrawerLayout from 'react-native-gesture-handler/ReanimatedDrawerLayout';
+```
+
+## Properties:
+
+### `drawerType`
+
+accepts values of the `DrawerPosition` enum, defaults to `DrawerPosition.FRONT`.  
+Specifies the way the drawer will be displayed.
+
+- `DrawerPosition.FRONT` the drawer will display on top of the content view.
+- `DrawerPosition.BACK` the drawer appears to be behind the content view, revealed by sliding away the content view.
+- `DrawerPosition.SLIDE` the drawer displays attached to the content view, opening it slides both the drawer and the content view.
```

Comment:
I'd use passive voice, i.e. _**drawer will be displayed**_ etc.

Reply:
Fixed in [ec0e598](https://github.com/software-mansion/react-native-gesture-handler/pull/3188/commits/ec0e598ee6dba06e004d23521e3e51caf01c28ee)